PowerEditor/installer/APIs/actionscript.xml

```diff
@@ -25,320 +22,211 @@
 		<KeyWord name="ALT" />
 		<KeyWord name="and" />
 		<KeyWord name="appendChild" />
-		<KeyWord name="appendChild" />
 		<KeyWord name="apply" />
 		<KeyWord name="arguments" />
 		<KeyWord name="arguments.callee" />
 		<KeyWord name="arguments.caller" />
 		<KeyWord name="Array" />
-		<KeyWord name="Array" />
 		<KeyWord name="AsBroadcaster" />
 		<KeyWord name="ASconstructor" />
 		<KeyWord name="asin" />
-		<KeyWord name="asin" />
 		<KeyWord name="ASnative" />
 		<KeyWord name="ASSetNative" />
 		<KeyWord name="ASSetPropFlags" />
 		<KeyWord name="atan" />
-		<KeyWord name="atan" />
-		<KeyWord name="atan2" />
 		<KeyWord name="atan2" />
 		<KeyWord name="attachAudio" />
 		<KeyWord name="attachMovie" />
-		<KeyWord name="attachMovie" />
-		<KeyWord name="attachSound" />
 		<KeyWord name="attachSound" />
 		<KeyWord name="attachVideo" />
 		<KeyWord name="attributes" />
-		<KeyWord name="attributes" />
 		<KeyWord name="autoSize" />
 		<KeyWord name="background" />
 		<KeyWord name="backgroundColor" />
 		<KeyWord name="BACKSPACE" />
-		<KeyWord name="BACKSPACE" />
 		<KeyWord name="beginFill" />
 		<KeyWord name="beginGradientFill" />
 		<KeyWord name="blockIndent" />
 		<KeyWord name="bold" />
 		<KeyWord name="Boolean" />
-		<KeyWord name="Boolean" />
 		<KeyWord name="border" />
 		<KeyWord name="borderColor" />
 		<KeyWord name="bottomScroll" />
 		<KeyWord name="break" />
-		<KeyWord name="break" />
 		<KeyWord name="broadcastMessage" />
 		<KeyWord name="bullet" />
 		<KeyWord name="Button" />
 		<KeyWord name="call" />
-		<KeyWord name="call" />
 		<KeyWord name="callee" />
 		<KeyWord name="caller" />
 		<KeyWord name="Camera" />
 		<KeyWord name="capabilities" />
 		<KeyWord name="Capabilities" />

```

Comment:
What about this? Does camelcase matters?

Reply:
I'm honestly not sure. I ran a simple regex to find duplicate lines. I am planning on looking into this eventually if case matters.